### ISSUE TITLE
Training with GPU show low accuracy

### ISSUE THREAD

**Issue by StpMax** [mindsdb/mindsdb_native#280]
* Python version: 3.6
* Operating system: ubuntu
* Mindsdb version: last staging mindsdb/native/lightwood
* Additional info if applicable: GeForce GTX 660

On my work pc i have pretty old gpu. Some time before training with this gpu without option `['use_gpu']=False` was impossible, so i set it permanently in code. Today i removed that, then noticed i get very low accuracy for new predictors. I trained two predictors, one trained with `['use_gpu']=False`, other without options:
![image](https://user-images.githubusercontent.com/4068133/98225899-35c55600-1f4d-11eb-9c2f-e281b079abb0.png)
as you see, accuracy have big difference.

**Comment by paxcema** [comment]
I was unable to replicate from the GUI, I'll try from within Native and report back.

**Comment by StpMax** [comment]
This problem has gone from me after i update torch by `pip install torch==1.7.0+cpu torchvision==0.8.1+cpu torchaudio==0.7.0 -f https://download.pytorch.org/whl/torch_stable.html`. Just one strange: now i have accuracy for new `rental_price/home_rentals` predictor 88.7%. If you think it is ok - then issue should be closed.
![image](https://user-images.githubusercontent.com/4068133/98514353-387fbe00-2261-11eb-9e4c-57d7530c9770.png)

**Comment by paxcema** [comment]
I'm getting good results, and could not replicate from Native staging either. I would guess this is related to the GPU model, so I think we should close this for now.

![image](https://user-images.githubusercontent.com/12536303/98550218-98cf2980-227a-11eb-976e-80831eaca0ac.png)
